File: local/HDR__/python/factorial.py
def factorial(n):
    result = 1
    for i in range(1, n + 1):
        result *= i
    return result

# result = []
# for i in range(1, 100 + 1):
#   result.append(factorial(i)) 

#result = [factorial(i) for i in range(1, 100)]

result = list(map(factorial, range(1,100 + 1))  )
print(result)

File: local/HDR__/c_src/factorial.c
#include <stdio.h>

long long factorial(int a)
{
	long long fact = 1;
	for (int i = 1; i<=a; ++i){
		fact *= i;
	}
	return fact;
}

int main(void)
{
	for (int i = 1; i <= 100; ++i){
		long long result = factorial(i);
		printf("%d! : %lld\n", i, result);
	}

	return 0;
}
